components/text_input/sample.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
+    <title>Text Input</title>
+  </head>
+  <body>
+
+    <form>
+      First Name:<br>
```

Comment:
replace with `<label>First Name:</label>`

Reply:
this may make the labels show to the left of the input field, if that occurs one way around it is to surround
the label, input into a div and flex it.

Reply:
Done